### PR TITLE
Plugin readme improvement

### PR DESCRIPTION
**Pull request by voodoologic**
I've been recently attempting to leverage nnn.  I think this addition to the plugin readme would help people who are starting out.

### CODE REVIEW

File: plugins/README.md [hunk 1]
```diff
@@ -94,6 +94,23 @@ To pick and run an unassigned plugin, press <kbd>Enter</kbd> (to _enter_ the plu
...
+```
+NNN_PLUG_PERSONAL='g:personal/convert2zoom;p:personal/echo'
+NNN_PLUG_WORK='j:work/prettyjson;d:work/foobar'
+NNN_PLUG_INLINE='e:_go run $nnn*'
+NNN_PLUG_DEFAULT='1:bookmarks;2:ipinfo;p:preview-tui;o:fzz;b:nbak'
+NNN_PLUG="$NNN_PLUG_PERSONAL;$NNN_PLUG_WORK;$NNN_PLUG_DEFAULT;$NNN_PLUG_INLINE"
+export NNN_PLUG
+```
+
+Two things stand out in the above example:
```
Comment (jarun) [inline plugins/README.md:108]: Please make this:

Note:

File: plugins/README.md [hunk 2]
```diff
@@ -94,6 +94,23 @@ To pick and run an unassigned plugin, press <kbd>Enter</kbd> (to _enter_ the plu
...
+- It will honor a path to a sub-folder.
```
Comment (jarun) [inline plugins/README.md:109]: Instead of "It", please mention the variable which does this.

File: plugins/README.md [hunk 3]
```diff
@@ -94,6 +94,23 @@ To pick and run an unassigned plugin, press <kbd>Enter</kbd> (to _enter_ the plu
...
+- Only the first defined keybindng will work, any subsequent keybinding defined on the same key won't work.
```
Comment (jarun) [inline plugins/README.md:110]: We can omit this as it's elementary.

Comment (jarun) [reply plugins/README.md:110]: Without this there will be only one note so probably you can add a single line note.

**Review by jarun — CHANGES_REQUESTED** [review]

**Comment by voodoologic** [comment]
Modified for your consideration.

**Comment by jarun** [comment]
Thank you!

**Pull request merged**